python/mujinwebstackclient/controllerwebclientraw.py

```diff
@@ -125,6 +153,10 @@ def Request(self, method, path, timeout=5, headers=None, **kwargs):
             kwargs['allow_redirects'] = method in ('GET',)
 
         response = self._session.request(method=method, url=url, timeout=timeout, headers=headers, **kwargs)
+        jsonWebToken = response.cookies.get('jwttoken')
```

Comment:
@LLipter You can also handle this part in the custom auth class with a response hook